flaskutils/models.py

```diff
@@ -1,8 +0,0 @@
-from pgsqlutils.orm import BaseModel
-
-
-class FlaskModel(BaseModel):
```

Comment:
Not, only that, all microservices will required an important refactoring but it will happen gradually